e2e/bundle-size/scenarios/vue-router-full/src/routes/__root.tsx

```
@@ -0,0 +1,191 @@
+  ScrollRestoration,
+  createLink,
+  createRootRoute,
+  linkOptions,
+  useAwaited,
+  useBlocker,
+  useCanGoBack,
+  useElementScrollRestoration,
+  useLinkProps,
+  useLoaderData,
+  useLoaderDeps,
+  useLayoutEffect,
```

ESLint

[error] Member 'useLayoutEffect' of the import declaration should be sorted alphabetically.

(sort-imports)